dependencies.list

```diff
@@ -1,4 +1,4 @@
 VERSION=2.0.0
-REALM_CORE_VERSION=2.0.0
+REALM_CORE_VERSION=2.0.1
```

Comment:
Hmm, I'm not sure this is even _used_ at this point. It looks like it's only used by `build.sh download-core`, and we've replaced all calls to this with `build.sh download-sync`.